state_machine/drivers/pycubedmini/lib/logging.py

```diff
@@ -66,30 +68,39 @@ def write_infotxt(logfile_starttime, folder=DEFAULT_FOLDER):
     info_file.close()
 
 
-def get_logfile_name_dir(logfile_starttime, folder=DEFAULT_FOLDER):
+def get_logfile_name_dir(logfile_starttime,
+                         folder=DEFAULT_FOLDER,
+                         time_interval=TIME_INTERVAL,
+                         file_name_interval=FILE_NAME_INTERVAL):
     logfile_name = get_logfile_name(logfile_starttime, folder)
     return f"{sd_card_directory}/{folder}/logs/{logfile_name}"
```

Comment:
Does circuitpython provide a `join` function for pathnames like `os.path.join`? I think that would be better to use here.

Reply:
the normal os library does have this, but the circuitpy supported version unfortunately doesn't

Reply:
i can define a helper function to do it and then call that (that could avoid potential future bugs)